test/spec_03_tests.ts

```diff
@@ -46,8 +46,13 @@ describe("CloudEvents Spec v0.3", () => {
 
   describe("OPTIONAL Attributes", () => {
     it("Should have 'datacontentencoding'", () => {
-      cloudevent.datacontentencoding = Constants.ENCODING_BASE64;
+      cloudevent = cloudevent.cloneWith({
+        datacontentencoding: Constants.ENCODING_BASE64,
+        data: "SSB3YXMgZnVubnkg8J+Ygg==",
```

Comment:
Same here, can you post the original string, then `btoa` it?

Reply:
There are other tests that already have the string encoded,  so the precedent was already set.  If this is a blocker, then we can create another PR to update the other tests as well

Reply:
Maybe add a TODO if other PRs are doing this?
It's still not ideal as I can't read or understand SSB3...